lib/galaxy_test/api/test_tags.py

```diff
@@ -155,12 +155,10 @@ def create_item(self) -> str:
 
         title = f"Test Visualization {uuid_str}"
         slug = f"test-visualization-{uuid_str}"
-        config = json.dumps(
-            {
-                "x": 10,
-                "y": 12,
-            }
-        )
+        config = {
```

Comment:
I don't think this change is correct, this makes me think the type of config used to be a JSON string and not a dictionary. And the UI and other API consumers might still expect that.

Reply:
I checked these ones and it seems that the config should be a dict:
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/base/controller.py#L709
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/api/visualizations.py#L383

Reply:
Hmm... ok, maybe the test itself was incorrect or the post without "json=true" was doing some conversion magic :thinking: